### PR TITLE
boac-5130 - Profile View Complete

### PR DESCRIPTION
**Pull request by mohamalnadi**
Jira: https://jira-secure.berkeley.edu/browse/BOAC-5130

### CODE REVIEW

File: src/components/admin/DemoModeToggle.vue [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Demo
```
Comment (pauline2k) [inline src/components/admin/DemoModeToggle.vue]: @mohamalnadi Are these title rows outside the template a new convention for Vue 3 files, or were they meant to be temporary?

File: src/components/admin/DemoModeToggle.vue [hunk 2]
```diff
@@ -10,23 +12,26 @@
...
             v-if="!isToggling"
             id="toggle-demo-mode"
             v-model="inDemoMode"
             :disabled="isToggling"
+            :label="`${inDemoMode ? 'On' : 'Off'}`"
             @change="toggle"
           >
-            <span class="sr-only">Demo mode is </span>{{ inDemoMode ? 'On' : 'Off' }}
-          </b-form-checkbox>
+            <!-- <span class="sr-only">Demo mode is </span>{{ inDemoMode ? 'On' : 'Off' }} -->
```
Comment (pauline2k) [inline src/components/admin/DemoModeToggle.vue]: Commented-out screenreader text: intentional?

**Comment by pauline2k** [comment]
Thanks @mohamalnadi! Looks good to me, a couple questions on specific lines. (Maybe @lyttam can take a look too since she's been closer to this work.)

File: src/components/admin/DemoModeToggle.vue (hunk 2 above)
Comment (lyttam) [reply src/components/admin/DemoModeToggle.vue]: I think we want either a `<label>` element separate from the `<v-checkbox>` _or_ `label` and `aria-label` properties on the `<v-checkbox>`. In the past I've noticed that using both a separate label and label properties causes the screen reader to voice it twice.

If you go with the second option, the `aria-label` property can be set to `Demo mode is ${inDemoMode ? 'On' : 'Off'}`.

**Comment by mohamalnadi** [comment]
@pauline2k @lyttam - Made the new changes! Thank you! 

I am not sure where the word "Demo" came from outside of the template, I may have added it accidentally?

For the Demo On/Off label, I added an aria-label. The reason I removed the span is because in Vuetify, the nested span text aligned to the left of the checkbox, but you would have to use a label to have the text on the right of the checkbox.

**Comment by lyttam** [comment]
Looks good, thanks!

**Pull request merged**